controllers/job.go

```diff
@@ -65,16 +65,21 @@ func unpackContainer(id, containerImage, pullImage string, pullOptions buildv1al
 	}
 }
 
-func createImageContainer(containerImage string, pushOptions buildv1alpha1.Push) v1.Container {
+func createImageContainer(containerImage string, artifact buildv1alpha1.OSArtifact) v1.Container {
+	imageName := "dontpush" // No image was defined, use a dummy one to let luet work
```

Comment:
that works, however, when loading the image then this name would be actually used

Reply:
changed the code to use `Spec.PushOptions.ImageName` when it's set, otherwise use the name of the artifact resource.